docs/lib/ssr/fragments/js/getting-started.md

```diff
@@ -0,0 +1,106 @@
+To work well with server-rendered pages, Amplify JS requires slight modifications from how you would use it in a client-only environment.
+
+## Amplify
+
+### Enabling SSR 
```

Comment:
Should we also mention that to make Auth SSR aware, they should use `withSSRContext` to sign in the user?

Also, is there any way to make a user authenticated via the UI components SSR aware?

Reply:
`withSSRContext` shouldn't be necessary on the client (and will impact bundle-sizes), but also has limited support for only `Auth`, `API`, and `DataStore` today.

UI components should be SSR-aware by default, since they defer to `Auth` that's been configured with `ssr: true`.